cli.js

```diff
@@ -2,19 +2,26 @@
 'use strict';
 const meow = require('meow');
 const opn = require('opn');
+const getStdin = require('get-stdin');
+const tempWrite = require('temp-write');
+const fileType = require('file-type');
 
 const cli = meow(`
 	Usage
 	  $ opn <file|url> [--wait] [-- <app> [args]]
+	  $ stdout | opn [--wait] [--ext] [-- <app> [args]]
 
 	Options
 	  --wait  Wait for the app to exit
+	  --ext   File extension for stdin
 
 	Examples
 	  $ opn http://sindresorhus.com
 	  $ opn http://sindresorhus.com -- firefox
 	  $ opn http://sindresorhus.com -- 'google chrome' --incognito
 	  $ opn unicorn.png
+	  $ echo ./image.png | opn
```

Comment:
I meant raw image data (that uses the `file-type` lib)

Not sure if this will work though so should be tested

```
$ cat ./image.png | opn
```